### ISSUE TITLE
Release Pillow 8.3.0 on July 1, 2021

### ISSUE THREAD

**Issue by hugovk** [python-pillow/Pillow#5413]
Add release notes

* [x] https://github.com/python-pillow/Pillow/pull/5416
* [x] https://github.com/python-pillow/Pillow/pull/5455
* [x] https://github.com/python-pillow/Pillow/pull/5402

## Main Release

Released quarterly on January 2nd, April 1st, July 1st and October 15th.

* [x] Open a release ticket e.g. https://github.com/python-pillow/Pillow/issues/3154
* [x] Develop and prepare release in `master` branch.
* [x] Check [GitHub Actions](https://github.com/python-pillow/Pillow/actions) and [AppVeyor](https://ci.appveyor.com/project/python-pillow/Pillow) to confirm passing tests in `master` branch.
* [x] Check that all of the wheel builds [Pillow Wheel Builder](https://github.com/python-pillow/pillow-wheels) pass the tests in Travis CI and GitHub Actions.
* [x] In compliance with [PEP 440](https://www.python.org/dev/peps/pep-0440/), update version identifier in `src/PIL/_version.py`
* [x] Update `CHANGES.rst`.
* [x] Run pre-release check via `make release-test` in a freshly cloned repo.
* [x] Create branch and tag for release e.g.:
  ```bash
  git branch 8.3.x
  git tag 8.3.0
  git push --all
  git push --tags
  ```
* [x] Create and check source distribution:
  ```bash
  make sdist
  twine check dist/*
  ```
* [x] Create [binary distributions](https://github.com/python-pillow/Pillow/blob/master/RELEASING.md#binary-distributions)
* [x] Check and upload all binaries and source distributions e.g.:
  ```bash
  twine check dist/*
  twine upload dist/Pillow-8.3.0*
  ```
* [x] Publish the [release on GitHub](https://github.com/python-pillow/Pillow/releases)
* [x] In compliance with [PEP 440](https://www.python.org/dev/peps/pep-0440/), increment and append `.dev0` to version identifier in `src/PIL/_version.py`

### Windows
* [x] Contact `@cgohlke` for Windows binaries via release ticket e.g. https://github.com/python-pillow/Pillow/issues/1174.
* [x] Download and extract tarball from `@cgohlke` and copy into `dist/`

### Mac and Linux
* [x] Use the [Pillow Wheel Builder](https://github.com/python-pillow/pillow-wheels):
  ```bash
  git clone https://github.com/python-pillow/pillow-wheels
  cd pillow-wheels
  ./update-pillow-tag.sh [[release tag]]
  ```
* [x] Download wheels from the [Pillow Wheel Builder release](https://github.com/python-pillow/pillow-wheels/releases)
  and copy into `dist/`

## Publicize Release

* [x] Announce release availability via [Twitter](https://twitter.com/pythonpillow) e.g. https://twitter.com/PythonPillow/status/1013789184354603010

## Documentation

* [x] Make sure the [default version for Read the Docs](https://pillow.readthedocs.io/en/stable/) is up-to-date with the release changes

## Docker Images

* [x] Update Pillow in the Docker Images repository
  ```bash
  git clone https://github.com/python-pillow/docker-images
  cd docker-images
  ./update-pillow-tag.sh [[release tag]]
  ```

**Comment by radarhere** [comment]
Hi @cgohlke, could we have Windows binaries for 8.3.0?

**Comment by cgohlke** [comment]
[Here](https://www.lfd.uci.edu/~gohlke/download/Pillow-8.3.0-win-dist.zip) you go.

I had to disable feature.raqm, otherwise _imagingft.pyd would depend on raqm.dll on my system.

**Comment by nulano** [comment]
> I had to disable feature.raqm, otherwise _imagingft.pyd would depend on raqm.dll on my system.

To solve some runtime linking issues with wheels on Linux, in #5062 I proposed some changes that turned out to be difficult to compile from source with pip (because Raqm is C99, which is incompatible with GCC defaults on some systems). So these changes should only be enabled when building **release** wheels, which requires building with the new `--global-option --vendor-raqm --global-option --vendor-fribidi` flags as seen here: https://github.com/python-pillow/pillow-wheels/pull/181#discussion_r550904961 Those prevent _imagingft.pyd from having a dependency on raqm.dll, and it should instead link fribidi.dll at runtime.

**Comment by radarhere** [comment]
Thanks @cgohlke!

[Released](https://pypi.org/project/Pillow/8.3.0/), and [announced!](https://twitter.com/PythonPillow/status/1410517213597999106)

**Comment by aclark4life** [comment]
And pinned!

<img width="1792" alt="Screen Shot 2021-07-01 at 5 30 24 AM" src="https://user-images.githubusercontent.com/72164/124101601-85e9af00-da2d-11eb-98b3-6275c4ac56b2.png">

**Comment by hugovk** [comment]
Looks like we need an 8.3.1 patch release: https://github.com/python-pillow/Pillow/issues/5571 is affecting lots who use NumPy either directly or indirectly (like torchvision), and there's a PR at https://github.com/python-pillow/Pillow/pull/5572.

And separately, Windows wheels would benefit from a new dependency, so a patch release would help there: https://github.com/python-pillow/Pillow/issues/5573#issuecomment-872691741

Is there anything else urgent that warrants inclusion in 8.3.1?

**Comment by radarhere** [comment]
There are some flags in https://github.com/python-pillow/Pillow/issues/5413#issuecomment-871978953 that cgohlke needs to apply to releases to resolve #4859. Not urgent, but something to flag for the next release.

**Comment by wiredfool** [comment]
On the bright side, I'm about 1.5 core months into a fuzzing run on a 8.3 near prerelease and nothing has come up.

**Comment by radarhere** [comment]
It would be good to include fixes for #5580 and #5583

**Comment by hugovk** [comment]
> It would be good to include fixes for #5580 and #5583

Agreed, let's include those regression fixes. Thanks for the PRs, I've merged them to master. 

I think we're ready to do 8.3.1? @radarhere Can you do it? If not, I can do it this evening (Europe time).

**Comment by radarhere** [comment]
Sure, I can do it. Closing. Future discussion can be in #5578
